qbnb/cli.py

```diff
@@ -0,0 +1,25 @@
+from qbnb import models
+from qbnb.models import update_listing
+from datetime import datetime
+
+
+def listing_update_page(listing):
+    new_title = input('Please enter new title: ')
+    new_description = input('Please enter new description: ')
+    new_price = float(input('Please enter new price: '))
+    if new_title == "":
+        new_title = None
+    if new_description == "":
+        new_description = None
+    if new_price == "":
```

Comment:
new_price will never equal an empty string because your input has been cast to a float.

Reply:
Fix has been implemented. Please review.